packages/backend/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { initIsEqual } from '@nordcraft/ssr/dist/rendering/equals'
+import { Hono } from 'hono'
+import type { HonoEnv } from '../hono'
+import { pageLoader } from './middleware/pageLoader'
+import { loadProjectInfo } from './middleware/projectInfo'
+import { routesLoader } from './middleware/routesLoader'
+import { proxyRequestHandler } from './routes/apiProxy'
+import { customElement } from './routes/customElement'
+import { favicon } from './routes/favicon'
+import { fontRouter } from './routes/font'
+import { manifest } from './routes/manifest'
+import { nordcraftPage } from './routes/nordcraftPage'
+import { robots } from './routes/robots'
+import { serviceWorker } from './routes/serviceWorker'
+import { sitemap } from './routes/sitemap'
+
+// Inject isEqual on globalThis
+// this is currently used by some builtin formulas
+initIsEqual()
+
+const app = new Hono<HonoEnv>()
+
+app.get('/sitemap.xml', loadProjectInfo, routesLoader, sitemap)
+app.get('/robots.txt', loadProjectInfo, robots)
+app.get('/manifest.json', loadProjectInfo, manifest)
+app.get('/favicon.ico', loadProjectInfo, favicon)
+app.get('/serviceWorker.js', loadProjectInfo, serviceWorker)
+
+// Nordcraft specific endpoints/services on /.toddle/ subpath 👇
+app.route('/.toddle/fonts', fontRouter)
+// Proxy endpoint for Nordcraft APIs
+app.all(
+  '/.toddle/omvej/components/:componentName/apis/:apiName',
+  proxyRequestHandler,
+)
+app.get(
+  '/.toddle/custom-element/:filename{.+.js}',
+  loadProjectInfo,
+  customElement,
+) // project infor + single component
+
+app.get(
+  '/*',
+  routesLoader,
+  loadProjectInfo,
+  // First we try loading a route if it exists
+  // routeLoader,
+  // createMiddleware<HonoEnv<HonoRoute & HonoRoutes>>((ctx, next) => {
+  //   const route = ctx.var.route
+  //   if (route) {
+  //     return routeHandler(ctx, route)
+  //   }
+  //   return next()
+  // }),
```

Comment:
Is there a plan for these lines of commented code?

Reply:
Yes. Currently, the worker doesn't support routes. But it should eventually 😅

Reply:
Nvm, I fixed it so it now supports routes 👍